analyzer/runtime/runtime.go

```diff
@@ -204,7 +203,7 @@ func (m *Main) prework() error {
 	// Register special addresses.
 	batch.Queue(
 		m.qf.AddressPreimageInsertQuery(),
-		rewards.RewardPoolAddress.String(),          // oasis1qp7x0q9qahahhjas0xde8w0v04ctp4pqzu5mhjav for emerald on mainnet oasis-3
+		rewards.RewardPoolAddress.String(),          // oasis1qp7x0q9qahahhjas0xde8w0v04ctp4pqzu5mhjav on mainnet oasis-3
```

Comment:
this is guaranteed to be the same for all current+future runtimes?

Reply:
looks like it. Comes from [here](https://github.com/oasisprotocol/oasis-sdk/blob/b37e6da699df331f5a2ac62793f8be099c68469c/client-sdk/go/modules/rewards/types.go#L10-L10). I guess the original comment here was misguided.